deploy/internal/service-mgmt.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: SYSNAME-mgmt
   labels:
     app: noobaa
+    noobaa-mgmt: "true"
```

Comment:
maybe use a different key for the label as we already use `noobaa-mgmt` for the pod labels. will be less confusing.  maybe `moobaa-mgmt-svc: "true"`

Reply:
done